Kbs.Wpf/User/Update/AccountView.xaml.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using Kbs.Business.User;
+using Kbs.Business.Session;
+using Kbs.Data.User;
+using Kbs.Wpf.Components;
+using Kbs.Wpf.User.Login;
+
+namespace Kbs.Wpf.User.Update
+{
+    /// <summary>
+    /// Interaction logic for AccountView.xaml
+    /// </summary>
+    public partial class AccountView : Page
+    {
+        private readonly UserValidator _userValidator = new UserValidator(); 
+        private readonly INavigationManager _navigationManager;
+        private readonly IUserRepository _userRepository = new UserRepository();
+
+        public AccountViewModel ViewModel => (AccountViewModel)DataContext;
+        public AccountView(INavigationManager navigationManager)
+        {
+            InitializeComponent();
+            this._navigationManager = navigationManager;
+            ViewModel.InputEmail = SessionManager.Instance.Current.User.Email;
```

Comment:
SessionManager.Instance.Current.User.Email is used several times throughout the file. Consider creating a local variable such as `var currentUser = SessionManager.Instance.Current.User.Email` to re-use it throughout the file